cmd/livepeer_cli/livepeer_cli.go

```diff
@@ -87,23 +93,76 @@ func (w *wizard) run() {
 	w.stats(false)
 	// Basics done, loop ad infinitum about what to do
 	for {
+
 		fmt.Println()
 		fmt.Println("What would you like to do? (default = stats)")
 		fmt.Println(" 1. Get node status")
-		fmt.Println(" 2. Deposit token")
-		fmt.Println(" 3. Broadcast video")
-		fmt.Println(" 4. Stream video")
-		fmt.Println(" 5. Set transcoder config")
-		fmt.Println(" 6. Set broadcast config")
-		fmt.Println(" 7. Bond")
-		fmt.Println(" 8. Unbond")
-		fmt.Println(" 9. Withdraw bond")
-		fmt.Println(" 10. Become a transcoder")
-		fmt.Println(" 11. Get test Livepeer Token")
-		fmt.Println(" 12. Get test Ether")
-		fmt.Println(" 13. List registered transcoders")
+		var choice string
+
+		if ( w.transcoder ) {
+			fmt.Println(" 2. Set transcoder config")
+			fmt.Println(" 3. Become a transcoder")
+			fmt.Println(" 4. Get test Livepeer Token")
+			fmt.Println(" 5. Get test Ether")
+			fmt.Println(" 6. List registered transcoders")
+			choice = w.read()
+			switch {
```

Comment:
Can we make a const that represents the execution path, and then map the user's choice to that?  The mapping between numbers can get confusing later.

Something like - 
```
const (
  Deposit = iota
  Broadcast
  //etc
)